<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>666</width>
    <height>335</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <property name="locale">
   <locale language="English" country="Germany"/>
  </property>
  <widget class="QLabel" name="labelPort">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>40</y>
     <width>49</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Port</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelBaudRate">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>70</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Baud Rate</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelData">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>100</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Data Bits</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelStop">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>130</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Stop Bits</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelParity">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>160</y>
     <width>49</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Parity</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelFlow">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>190</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Flow Control</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxPort">
   <property name="geometry">
    <rect>
     <x>148</x>
     <y>40</y>
     <width>121</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxBaudRate">
   <property name="geometry">
    <rect>
     <x>148</x>
     <y>70</y>
     <width>121</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxData">
   <property name="geometry">
    <rect>
     <x>148</x>
     <y>100</y>
     <width>121</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxStop">
   <property name="geometry">
    <rect>
     <x>148</x>
     <y>130</y>
     <width>121</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxParity">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>160</y>
     <width>121</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxFlowControl">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>190</y>
     <width>121</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>40</y>
     <width>351</width>
     <height>192</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_Send">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>250</y>
     <width>261</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Send">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>250</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Send</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Connect">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>260</y>
     <width>211</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Connect</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Disconnect">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>300</y>
     <width>211</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Disconnect</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Refresh">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>300</y>
     <width>121</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Refresh Ports</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelCode">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>220</y>
     <width>49</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Code</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditCode">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>220</y>
     <width>121</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Clear">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>10</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Clear</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
